helm/templates/ingress_timecard.yml

```diff
@@ -11,6 +11,8 @@ metadata:
     ingress.kubernetes.io/force-ssl-redirect: "true"
     kubernetes.io/ingress.class: "nginx-internal"
     kubernetes.io/backend-protocol: "HTTPS"
+    ingress.kubernetes.io/enable-cors: "true"
+    ingress.kubernetes.io/cors-allow-origin: "https://{{ .Values.ui.host }}"
```

Comment:
Does the UI port need to be included here? Or not needed?

Reply:
Good shout. I think it's fine without the port - since the API request originates in the UI running on a standard port (but I could be wrong!)

Reply:
Ah ok should be fine then if its on a standard port.